packages/vue/src/components/molecules/SfModal/SfModal.js

```diff
@@ -48,49 +52,38 @@ export default {
   },
   methods: {
     close() {
-      this.$emit("close");
-      // remove left code
-    },
-    keydownHandler(e) {
-      if (e.key === "Escape" || e.key === "Esc" || e.keyCode === 27) {
-        this.close();
-      }
-      if (this.visible) {
-        // TODO: this is an error right now. what shall it do?
-        //  document.addEventListener("keydown", keydownHandler);
-        this.$emit("close");
-      } else {
-        // TODO: this is an error right now. what shall it do?
-        //  document.removeEventListener("keydown", keydownHandler);
-        this.$emit("open");
-      }
+      this.$emit("toggle", false);
     },
     checkPersistence() {
-      if (this.persistent === false) {
+      if (!this.persistent) {
         this.close();
       }
     }
   },
   watch: {
     visible: {
-      handler: function(visibility) {
-        if (visibility && typeof window !== "undefined") {
-          document.body.style.setProperty(
-            "margin-right",
-            `${window.innerWidth - document.body.clientWidth}px`
-          );
-          document.addEventListener("keydown", this.keydownHandler);
+      handler: value => {
+        if (typeof window === "undefined") return;
+        if (value) {
           document.body.style.setProperty("overflow", "hidden");
-        }
-        if (!visibility && typeof window !== "undefined") {
-          document.body.style.removeProperty("margin-right");
+        } else {
           document.body.style.removeProperty("overflow");
-          document.removeEventListener("keydown", this.keydownHandler);
         }
       },
       immediate: true
     }
   },
+  created() {
```

Comment:
```suggestion
  mounted() {
```
please use "always" mounted() because created run on the server when you use SSR :), mounted was trigger only on client side